deepfence_ui/app/scripts/components/compliance-view/summary-cis.js

```diff
@@ -26,14 +25,6 @@ const testValueConfig = [
 ];
 
 class CISSummary extends React.PureComponent {
-
-  componentDidMount() {
-    const cloudType = window.location.hash.split('/').reverse()[3];
-    const checkType = window.location.hash.split('/').reverse()[0];
-    const nodeId = window.location.hash.split('/').reverse()[2];
-    this.props.dispatch(getComplianceChartDataAction({nodeId, checkType, cloudType}));
```

Comment:
this.props.dispatch(getComplianceChartDataAction) is not needed anymore?

Reply:
yes, it was not necessary, it is called in the child components anyways.